src/Mvc/src/Microsoft.AspNetCore.Mvc.NewtonsoftJson/NewtonsoftJsonHelper.cs

```diff
@@ -17,39 +17,37 @@ namespace Microsoft.AspNetCore.Mvc.NewtonsoftJson
     /// </summary>
     internal class NewtonsoftJsonHelper : IJsonHelper
     {
-        private readonly NewtonsoftJsonOutputFormatter _jsonOutputFormatter;
-        private readonly ArrayPool<char> _charPool;
+        private readonly JsonSerializer _defaultSettingsJsonSerializer;
```

Comment:
@JamesNK are the methods of `JsonSerializer` thread-safe? (Elsewhere we use `ObjectPool<JsonSerializer>` but that may only be because we add / remove event handlers.)

Reply:
This random person here on SO seems to think so - https://stackoverflow.com/a/36189439

Reply:
It is fine to do this. Add a comment so the question doesn't come up again: https://github.com/aspnet/AspNetCore/blob/36529e4a344167692f30f8ca21c2040dc38a388e/src/Mvc/src/Microsoft.AspNetCore.Mvc.NewtonsoftJson/NewtonsoftJsonOutputFormatter.cs#L22-L24